src/main/java/org/scoverage/plugin/SCoverageReportMojo.java

```diff
@@ -547,6 +551,32 @@ else if ( !module.getPackaging().equals( "pom" ) )
         getLog().info( "Coverage aggregated reports completed." );
     }
 
+    private MavenProject findAncestorModuleWithShortestBasedir()
```

Comment:
seems like this is not really related to minimums we can set, right?
seems like it's similar to https://github.com/scoverage/scoverage-maven-plugin/pull/104
could you please clarify why we need this here?  Can we extract that to a separate PR?

Reply:
ah, correct. the code is what we used in our production for many years, and it's likely that there were two unrelated changes to make everything work. will remove from this pr (and the linked one might be a better implementation).

Reply:
Just a heads-up, i've merged #104, so dropping this particular change should be safe now.